Library/ViewModels/SettingsViewModel.swift

```diff
@@ -533,6 +540,12 @@ SettingsViewModelOutputs {
   public func postLikesTapped(selected: Bool) {
     self.postLikesTappedProperty.value = selected
   }
+
+  fileprivate let showPublicProfileProperty = MutableProperty(false)
+  public func privateProfileSwitchDidChange(isOn: Bool) {
+    self.showPublicProfileProperty.value = !isOn
```

Comment:
I think we should only perform transformation on the boolean in the output signal. These inputs are meant to have no implementation detail and that way we also know where to look for any transformation that we may be performing on the value.

Reply:
Not sure what's up with the double review 😄

Reply:
I moved the transformation to line 186:

```
self.privateProfileEnabledProperty.signal.negate().map {
    (UserAttribute.privacy(Privacy.showPublicProfile), $0)
}
```

The issue is that it needs to be transformed before it gets stored on the `User` object since variable name has the opposite meaning: `showPublicProfile` (on the User object) vs. `privateProfileEnabled` (user facing text). Does it make more sense to have it there?

Reply:
Yip that's good 👍 We know that all of our transformations are happening in our output signals so that's where we'll look for it in the future.

I have one more question, I'll ask it on the relevant line.